sky/serve/controller.py

```diff
@@ -98,12 +98,14 @@ def run(self) -> None:
 
         @self._app.post('/controller/load_balancer_sync')
         async def load_balancer_sync(
-                request: fastapi.Request) -> fastapi.Response:
-            request_data = await request.json()
-            # TODO(MaoZiming): Check aggregator type.
-            request_aggregator: Dict[str, Any] = request_data.get(
-                'request_aggregator', {})
-            timestamps: List[int] = request_aggregator.get('timestamps', [])
+                request: schemas.LoadBalancerRequest) -> fastapi.Response:
+            request_aggregator = request.request_aggregator
```

Comment:
```suggestion
            request_aggregator_info = request.request_aggregator_info
```
lets distinguish aggregator and the information they produced